doc/2/plugins/plugin-context/accessors/sdk/index.md

```diff
@@ -69,13 +69,72 @@ async myAwesomePipe (request) {
 
 - The created document will have the `author` metadata property set to `null`.
 
+## Realtime notifications
+
+<SinceBadge version="change-me" />
+
+The [realtime](/sdk/js/7/controllers/realtime) is entirely available with the [subscribe](/sdk/js/7/controllers/realtime/subscribe) and [unsubscribe](/sdk/js/7/controllers/realtime/unsubscribe) methods.  
+
+Realtime subscription should be made in the plugin [init](core/2/plugins/guides/manual-setup/init-function) method or in a hook on the [kuzzle:start:before](/core/2/plugins/guides/events/kuzzle-start) event.
+
+::: warning
+If you want to make subscription in hooks, pipes, or controller actions then you should use the `replicate: true` option otherwise your callback will not be executed with every notification.
+:::
+
+When you receive a notification, only one node will execute the associated callback.  
+
+::: info
+  You can use the `replicate` option if you want your callback function to be executed on each nodes.
+:::
+
+#### replicate: false (default)
+
+With `replicate: false`, the callback function will be executed only on the node receiving a request who triggers a notification. (only one execution) 
```

Comment:
Impractical: subscriptions should always be kept in sync in all nodes. The question is if a notification is propagated/duplicated or not. Am I right?